x/metadata/keeper/scope_test.go

```diff
@@ -1370,6 +1401,41 @@ func (s *ScopeKeeperTestSuite) TestValidateScopeUpdateOwners() {
 			signers:  []string{s.user1, s.user2},
 			errorMsg: "",
 		},
+		{
+			name:     "smart contract without provenance role added",
+			existing: scopeWithOwners(ownerPartyList(s.user1)),
+			proposed: scopeWithOwners(ownerPartyList(s.user1, s.scUser)),
+			signers:  []string{s.user1},
+			errorMsg: `account "` + s.scUser + `" is a smart contract but does not have the PROVENANCE role`,
+		},
+		{
+			name:     "smart contract without provenance role removed",
```

Comment:
Can't wrap my head around if this test should have an error or not. I was initially thinking it should because every smart contract is supposed to have a Provenance Role, but it might be safer to allow removals.

Reply:
This one is basically checking that the validation is NOT run on the existing entry. E.g. if an existing scope has a smart contract in a party that isn't provenance, it can be updated to either remove it or switch it to provenance.